bokeh/models/map_plots.py

```diff
@@ -83,10 +83,6 @@ def _check_missing_renderers(self):
     def _check_no_glyph_renderers(self):
         pass
 
-    map_options = Instance(GMapOptions, help="""
-    Options for displaying the plot.
-    """)
-
```

Comment:
I think this shouldn't have been removed. Note the type difference with the other case below.